src/group/api/groups.js

```diff
@@ -72,6 +72,10 @@ export default {
   trustUser (groupId, userId) {
     return axios.post(`/api/groups/${groupId}/users/${userId}/trust/`)
   },
+
+  revokeTrust (groupId, userId) {
+    return axios.post(`/api/groups/${groupId}/users/${userId}/revoketrust/`)
```

Comment:
A `DELETE `/api/groups/${groupId}/users/${userId}/trust/` API call would seem more consistent with our other API endpoints (e.g. https://github.com/yunity/karrot-frontend/blob/5c6c6e8f2b845c230e6878f249bd6c2ba37efff8/src/messages/api/reactions.js#L21-L23) - which could be described as "_vaguely RESTful_".

Reply:
Oh yes very good point!